### ISSUE TITLE
disabling docker by default; removing it from the compose?

### ISSUE THREAD

**Issue by miabbott** [openshift/os#170]
In coreos/redhat-release-coreos#6 there was mention of either disabling `docker` by default or removing it completely from the compose.

Rather than litter that PR with design discussion, I've opened this issue.

While I'm in favor of both of these (maybe disable first, then incrementally move towards removal) the use case that immediately comes to mind is `oc cluster up` or `minishift`.  The `oc cluster up` case still requires `docker` (not sure about `minishift`).  I believe one or both of those were being considered as a development path for OpenShift devs?

So outright removal doesn't seem wise right now?

**Comment by ashcrow** [comment]
I agree. Outright removal right now wouldn't allow @crawford and others working on installer code to incrementally continue working towards goals.
